README.md

```diff
@@ -40,27 +28,36 @@ and we will work with you.
 **Officially supported browsers:** The last two versions of Chrome, Edge,
 Firefox, and Safari.
 
-## Package [build_runner] support
+## Package `build_runner` support
 
-Version 0.9.0-alpha+3 added experimental support for users of package
-build_runner. To build your project with build_runner:
+As of angular: 5.0.0-alpha+5 the pub transformer has been removed in favor of
+code generation through package [build]. To build your project with
+[build_runner]:
 
-1. Add two `dev_dependencies` to your packages pubspec.yaml file:
-  ```
+1. Add two dev dependencies to your `pubspec.yaml` file:
+
+  ```yaml
   dev_dependencies:
     build_runner: ^0.7.0
```

Comment:
I think that maybe this needs to be ^0.7.8?

Reply:
Done.